admin-assets/nginx/default.conf

```diff
@@ -5,6 +5,8 @@ server {
    # don't send the nginx version number in error pages and Server header
   server_tokens off;
 
+  error_log /dev/stdout debug;
```

Comment:
Just to confirm - is this wanted / required for ops or is this debug?

Reply:
this isnt part of this PR!  this has managed to get in when traversing branches last week